spec/models/casa_case_spec.rb

```diff
@@ -35,13 +35,20 @@
   end
 
   describe ".unassigned_volunteers" do
+    let!(:casa_case) { create(:casa_case) }
+    let!(:volunteer_same_org) { create(:volunteer, display_name: "Yelena Belova", casa_org: casa_case.casa_org) }
+    let!(:volunteer_same_org_1_with_cases) { create(:volunteer, :with_casa_cases, display_name: "Natasha Romanoff", casa_org: casa_case.casa_org) }
+    let!(:volunteer_same_org_2_with_cases) { create(:volunteer, :with_casa_cases, display_name: "Melina Vostokoff", casa_org: casa_case.casa_org) }
+    let!(:volunteer_different_org) { create(:volunteer, casa_org: create(:casa_org)) }
+
     it "only shows volunteers for the current volunteers organization" do
-      casa_case = create(:casa_case)
```

Comment:
please put these back, they're what prove the test is working :)

Reply:
I moved the create into a let so it can be used by both tests: `let!(:casa_case) { create(:casa_case) }`

Reply:
ah cool